packages/sdk/src/server/__tests__/server.test.ts

```
@@ -726,17 +727,22 @@
 
   beforeEach(() => {
     intercepts = [];
-    jest.spyOn(global, "fetch").mockImplementation(jest.fn((url: string, init: RequestInit) => {
+    // without these generics this fails typecheck and can't figure out why
+    jest.spyOn<any, any, any>(global, "fetch").mockImplementation(jest.fn<typeof fetch>(async (...args: Parameters<typeof fetch>) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)